pymc3/tests/test_examples.py

```diff
@@ -148,30 +153,138 @@ def too_slow(self):
 
 
 def build_disaster_model(masked=False):
-    disasters_data = np.array([4, 5, 4, 0, 1, 4, 3, 4, 0, 6, 3, 3, 4, 0, 2, 6,
-                               3, 3, 5, 4, 5, 3, 1, 4, 4, 1, 5, 5, 3, 4, 2, 5,
-                               2, 2, 3, 4, 2, 1, 3, 2, 2, 1, 1, 1, 1, 3, 0, 0,
-                               1, 0, 1, 1, 0, 0, 3, 1, 0, 3, 2, 2, 0, 1, 1, 1,
-                               0, 1, 0, 1, 0, 0, 0, 2, 1, 0, 0, 0, 1, 1, 0, 2,
-                               3, 3, 1, 1, 2, 1, 1, 1, 1, 2, 4, 2, 0, 0, 1, 4,
-                               0, 0, 0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 1, 0, 1])
+    disasters_data = np.array(
```

Comment:
Please use # fmt: off here, to disable (and then re-enable) Black formatting -- see step 3 of the [style guide](https://github.com/pymc-devs/pymc3/wiki/PyMC's-Jupyter-Notebook-Style)

Reply:
@Yash1256 thanks for updating - I think you'll want to checkout this file to how it is on upstream/master before putting `# fmt: off` and running `black` again.

Workflow would be something like
```
git fetch upstream --prune
git checkout upstream/master -- pymc3/tests/test_examples.py
<add in #fmt: off and #fmt: on as suggested by Alex>
black pymc3/tests/test_examples.py
git add -u
git commit -m <some descriptive message lol>
git push -u origin HEAD
```

Reply:
Please review @AlexAndorra @MarcoGorelli